lib/remap-tests/tests/invalid_path.vrl

```diff
@@ -0,0 +1,11 @@
+# object: { "bar": "bar" }
+# result: 
+#
+# remap error: parser error:  --> 2:6
+#   |
+# 2 | .bar.␊
```

Comment:
Surely?

```suggestion
# 2 | .foo.␊
```

Reply:
No. The object is `{ "bar": "bar" }`, so `.bar` is correct.

Now, if you had commented on the _other_ test, then you would have been correct because I had that test intentionally failing for now, for testing purposes. But that's no longer needed, so I'll update that one to pass as well 👍

Reply:
Is this not the error message it gets when parsing the code below, which is:

```coffeescript
.foo.
```

So the input object is irrelevant?

Reply:
Oh sorry, I missed that 😄 So yes, I apparently wanted both tests to fail at some point. I'll update the expectations before I merge the PR 👍

Reply:
(see also the screenshot in the [PR description](https://github.com/timberio/vector/pull/6022#issue-554146028), which shows this exact case erroring)